server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -284,6 +285,11 @@ def main():
         return 1
 
     logger = get_pbench_logger(_NAME_, config)
+    try:
+        logging_level = config.get(_NAME_, "logging_level")
+        logger.setLevel(logging_level)
```

Comment:
Typically, you use an `else` clause in this situation:

```
try:
    logging_level = config.get(_NAME_, "logging_level")
except (NoSectionError, NoOptionError):
    pass
else:
    logger.setLevel(logging_level)
```

Could we consider move this logging level setting into `get_pbench_logger` itself?

Reply:
We can discuss about this in scrum